src/main/java/frc/robot/Subsystem/Climb/ClimbSubsystem.java

```diff
@@ -26,37 +30,53 @@ public static ClimbSubsystem getInstance(){
 
   /** Creates a new ClimbSubsystem. */
   private ClimbSubsystem() {
-    m_rightMotor = new CANSparkMax(M_CLIMBINGRIGHT_MOTOR_ID, MotorType.kBrushless);
-    m_leftMotor = new CANSparkMax(M_CLIMBINGLEFT_MOTOR_ID, MotorType.kBrushless);
-    m_leftMotor.setInverted(false);
-    m_rightMotor.setInverted(true);
+    m_rightMotor = new CANSparkMax(M_CLIMBING_RIGHT_MOTOR_ID, MotorType.kBrushless);
+    m_leftMotor = new CANSparkMax(M_CLIMBING_LEFT_MOTOR_ID, MotorType.kBrushless);
+
+    configs();
+  }
+
+  public DoubleSupplier getSpeed()
+  {
+    return (() -> m_rightMotor.getAppliedOutput());
   }
   
   /**
    * set the speed of both motors
    * @param speed electrecity between -1 to 1 
    */
-  public void setSpeed(double speed){
-    m_rightMotor.set(speed);
-    m_leftMotor.set(speed);
+  public Command moveClimb(DoubleSupplier speed){
+    return run(() -> {
+      m_leftMotor.set(speed.getAsDouble());
+      m_rightMotor.set(speed.getAsDouble());
+    });
   }
 
   /**
-   * set the speed of the right motors
-   * @param speed electrecity between -1 to 1 
+   * Creates a command that makes the climb motors move with custom left and right speeds.
+   *
+   * @param  leftSpeed  a DoubleSupplier that provides the speed for the left motor
+   * @param  rightSpeed a DoubleSupplier that provides the speed for the right motor
+   * @return            a Command that moves the climb motors with the provided speeds
    */
-  public void setRightSpeed(double speed){
-    m_rightMotor.set(speed);
+  public Command moveClimb(DoubleSupplier rightMotorSpeed, DoubleSupplier leftMotorSpeed){
```

Comment:
comments have the params in reverse order than the order you get them in the () 

i would also advise to get the leftMotorSpeed first so it would be on the left